android/capacitor.settings.gradle

```diff
@@ -6,4 +6,4 @@ include ':capacitor-toast'
 project(':capacitor-toast').projectDir = new File('../node_modules/@capacitor/toast/android')
 
 include ':proofme-id-sdk'
-project(':proofme-id-sdk').projectDir = new File('../../../Libraries/Proofme-SDK/./web/reader/android')
+project(':proofme-id-sdk').projectDir = new File('../../Proofme-SDK/modules/reader/android-src')
```

Comment:
Doesn't this have to be commented out?

Reply:
Will be generated each time when you build android